CHANGELOG.md

```diff
@@ -41,8 +41,9 @@ Main (unreleased)
   - Add enable/disable collector configurability to `database_observability.mysql`. This removes the `query_samples_enabled` argument, now configurable via enable/disable collector. (@fridgepoet)
   - Refactor cache config in schema_table collector (@cristiangreco)
   - Use labels for some indexed logs elements (@cristiangreco)
+  - Correctly quote schema and table name in SHOW CREATE (@cristiangreco)
```

Comment:
Would it useful to include the component this impacts in the changelog? This probably also applies to previous entries we've made for this release

Reply:
Will do that, although I'm now thinking this long list is pretty much useless while the component is still experimental.